.github/workflows/ci-tests-n-coverage.yml

```diff
@@ -24,6 +24,19 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     needs: [integration-tests]
+    services:
+      postgres:
+        image: postgres:14
+        env:
+          POSTGRES_PASSWORD: 123456
```

Comment:
Should this be hidden? I don't think leaving it like this is a problem but I can be missing something here.

Reply:
> Should this be hidden? I don't think leaving it like this is a problem but I can be missing something here.

Maybe just leave it blank?